CONTRIBUTING.md

```diff
@@ -98,24 +99,33 @@ Contributors may provide database identifiers associated with each member of the
+
+
+## Source files
+The ``source`` directory for each community contains various information defined by the contributor. ``source/taxonomy.tsv``, described above, is required, as it should ideally contain strain information that may not be preserved in the database-specific ``expected-taxonomy.tsv`` files. Additionally, other files may be relevant, such as full-length 16S rRNA (or other marker-gene) sequences that the user sequenced from each strain. Naming conventions are not enforced for these optional files, but try to use short, descriptive names. As github enforces size limits on repositories, only small files are accepted ``source``, and may be removed by the mockrobiota developers at their personal discretion without notice to the contributor. Don't even try uploading whole genome sequences.
 
 ## Submitting to mockrobiota
 mockrobiota is hosted on [GitHub](http://www.github.com), and we use GitHub's [Pull Request](https://help.github.com/articles/using-pull-requests) mechanism for reviewing and accepting submissions. On submission of a pull request, a series of tests will be run to confirm the integrity of the submitted data (as well as to re-test the integrity of all existing data). We require these tests to pass for your data set before we will merge it to ensure the overall integrity of the mockrobiota resource.
 
+To test the integrity of your local branch before submitting a pull request, move to the ``mockrobiota`` home directory in terminal and type:
+``python tests/check_data_integrity.py``
+
+If no error messages are thrown, you are safe to proceed.
+
 ## Compiling expected taxonomy files
 
 Expected composition data will consist of one of the following types:
 
 * Marker-gene mock community: expected taxonomic composition for a mixture of microbial cells. The taxonomic annotations present in the expected data will be specific to the database version that is used for analysis, and will be meaningless if used for different database versions. Likewise, they may not match the source annotation (i.e., the taxonomy of each strain to the best knowledge of the mock community’s creator) if taxonomic annotations have been revised or if the reference database being used does not contain a given taxonomy.
-* Metagenome mock community: expected gene composition for a mixture of microbial cells/genomes. Gene annotations will be reference database specific, as for marker-gene mock communities above.
+* Metagenome mock community: expected gene composition for a mixture of microbial cells/genomes. Gene annotations will be reference database specific, as for marker-gene mock communities above. The expected taxonomic composition is also acceptable and encouraged, as described for marker-gene mock communities above. Note that the reference database used to annotate these will identify whether taxonomic information (e.g., ``greengenes/13-8/99-otus/``) or gene composition information (e.g., ``kegg/genes/100-otus/``) is present.
```

Comment:
> ``kegg/genes/100-otus/``

Wouldn't there still be a database version number here? I wonder if it should instead be something like ``kegg/<version-number>/genes``, so ``genes`` is taking the place of ``99-otus`` here, for example.